usaspending_api/references/tests/integration/filter_tree/psc/psc_data_fixtures.py

```diff
@@ -0,0 +1,138 @@
+from model_mommy import mommy
+import pytest
+
+
+@pytest.fixture
+def no_data(db):
```

Comment:
How is this different than simply providing `db` as a test function parameter?

Reply:
This ultimately comes down to "it made sense at the time". I have no opinions one way or the other on how to define this.